ClubMembersSortedCF/src/members/v1/InvalidZipCodeException.java

```diff
@@ -0,0 +1,7 @@
+package members.v1;
+
+public class InvalidZipCodeException extends Throwable{
+    public InvalidZipCodeException(String s) {
+        System.out.println(s);
+    }
+}
```

Comment:
Here is a more conventional way to write a custom exception:

`public class InvalidZipCodeException extends Exception {
    public InvalidZipCodeException(String message) {
        super(message);
    }
}`